github/event_types.go

```diff
@@ -438,6 +439,11 @@ type RepoName struct {
 	From *string `json:"from,omitempty"`
 }
 
+// EditTopics represents a change of repository topics.
+type EditTopics struct {
+	From *[]string `json:"from,omitempty"`
```

Comment:
Rarely do we actually need a pointer to a slice since a slice is already a reference type, and here we don't need one (and they are a pain in the rear to use), so let's please remove it:

```suggestion
	From []string `json:"from,omitempty"`
```

Reply:
Thanks for this suggestion, interesting reason I take in note for the future :+1: